smoke-test/cdep.yml

```diff
@@ -36,6 +36,8 @@ dependencies:
 - compile: com.github.jomof:stb/tilemap_editor:0.0.0-rev6
 - compile: com.github.jomof:stb/truetype:0.0.0-rev6
 - compile: com.github.jomof:stb/voxel_render:0.0.0-rev6
-# - compile: com.github.gpx1000:v8:6.3.2163
-- compile: com.github.gpx1000:zlib:1.2.11
+- compile: com.github.gpx1000:curl:7.56.2
```

Comment:
Is it intentional that you have curl twice? Here and above

Reply:
Not sure I understand this comment.  Where is the other curl?

Reply:
lgtm

Reply:
I'll have to remember to readd v8 when I get an armeabi-v7a build done (need to update it).

Reply:
Actually, what do you think about uncommenting boringssl and zlib? Both are useful standalone and this file sort of serves as a manifest of available components.

Reply:
I think we might want to talk about making a manifest in the docs.  While I like the idea of adding boringssl and zlib to the smoketest, they already are exercised from being a part of curl.
It feels like docs would be a better location for such a list as this is predominately an integration test component.  The manifest can also serve as a location for any parties to "Self add" and give project landing pages etc.  That said, I don't have strong feelings one way or the other; thus I'll defer to you.